docs/README.md

```
@@ -2,77 +2,44 @@
-## Exporting and creating packages
+Creating packages (i.e., creating an .exe with the game) and exporting them is allowed by YoYo Games for non-commercial use.\
+Everything else **must** be covered by [paid **GameMaker** subscriptions](https://gamemaker.io/en/get).
 
-Creating packages (i.e. creating an .exe with the game) and exporting them is allowed for non-commercial use.
+## Contributing
 
-Everything else **must** be covered by paid **GameMaker** subscriptions:
-- [GameMaker subscriptions comparison](https://gamemaker.io/en/get)
-- [Setting up GameMaker export for Windows](https://help.yoyogames.com/hc/en-us/articles/235186048-Setting-Up-For-Windows)
+Best bet is to ask about everything in our Discord, because things bellow are probably not very helpful at the moment.
+- [Contributing](https://github.com/Adeptus-Dominus/ChapterMaster/wiki/Contributing) wiki page (not filled properly)
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)